tests/python_test.py

```diff
@@ -53,9 +53,7 @@
 
 LOCKSPACE_OR_RESOURCE_NAMES = [
     # Bytes are supported with python 2 and 3.
-    pytest.param(
-        b"\xd7\x90",
-        marks=pytest.mark.xfail(six.PY3, reason="bytes support not implemented yet")),
+    pytest.param(b"\xd7\x90"),
```

Comment:
We don't need the pyparam() wrapper now, but we can keep it for symmetry if you like.

Reply:
i know